tests/caasp/oci_install.pm

```diff
@@ -8,15 +8,26 @@
 # without any warranty.
 
 # Summary: Start CaaSP installation
-# Maintainer: Martin Kravec <mkravec@suse.com>
+# Maintainer: Martin Kravec <mkravec@suse.com>, Panagiotis Georgiadis <pgeorgiadis@suse.com>
 
 use strict;
 use warnings;
 use base "y2logsstep";
+#use utils qw(handle_login handle_emergency get_x11_console_tty);
 use testapi;
 use caasp;
 
 sub run {
+    # Verify that workaround for Microfocus infobloxx GEO-IP DNS Cluster works
+    if (get_var('EDGECAST')) {
+        record_info 'Netfix', 'Go through Europe Microfocus info-bloxx';
+        my $edgecast_europe = get_var('EDGECAST');
+        select_console 'install-shell';
+        assert_script_run("ping -c 3 updates.suse.com");
+        assert_script_run("ping -c 1 updates.suse.com | grep $edgecast_europe");
+        assert_script_run('chvt 7');
```

Comment:
you're meant to select_console again

Reply:
what's the appropriate console to go back to the installation screen?

Reply:
installation - verify installation/logpackages